packages/storefront-lwc/src/modules/commerce/basket/__tests__/basket.test.js

```diff
@@ -0,0 +1,276 @@
+                        '    at resolveFieldValueOrError (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:467:18)',
+                        '    at resolveField (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:434:16)',
+                        '    at executeFields (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:275:18)',
+                        '    at executeOperation (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:219:122)',
+                        '    at executeImpl (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:104:14)',
+                        '    at Object.execute (/home/blittle/dev/sfcc-sample-apps/node_modules/graphql/execution/execute.js:64:35)',
+                        '    at /home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:241:48',
+                        '    at Generator.next (<anonymous>)',
+                        '    at /home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:8:71',
+                        '    at new Promise (<anonymous>)',
+                        '    at /home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:4:12',
+                        '    at execute (/home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:220:20)',
+                        '    at Object.<anonymous> (/home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:160:42)',
+                        '    at Generator.next (<anonymous>)',
+                        '    at fulfilled (/home/blittle/dev/sfcc-sample-apps/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)',
+                        '    at processTicksAndRejections (internal/process/task_queues.js:97:5)',
+                    ],
+                },
+            },
+        },
+    ],
+    data: null,
+};
+const basketWithItem = {
+    data: {
+        getBasket: {
+            basketId: 'cb60c056280a77d090d5a25c59',
+            customerId: 'bc5tfb0zfyMRyKrkXaxyha1Q3U',
+            getBasketMessage: '',
+            totalProductsQuantity: 1,
+            shipmentId: 'me',
+            shipmentTotal: 143.84,
+            selectedShippingMethodId: '001',
+            products: [
+                {
+                    productId: '701644031213M',
+                    itemId: '7f3b564f59e7cea7462a9e30a5',
+                    quantity: 1,
+                    productName: 'Floral Dress',
+                    price: 129,
+                    imageURL:
+                        'https://staging-functional07-qa.demandware.net/on/demandware.static/-/Sites-apparel-m-catalog/default/dw075478dd/images/small/PG.10237222.JJB52A0.PZ.jpg',
```

Comment:
This URL also can not pass in another environment.

Reply:
Removed it out as well